lightning_covid19/model/densenet.py

```diff
@@ -167,12 +195,26 @@ def prepare_data(self):
         else:
             print('Cloning the covid chestxray dataset. It may take a while\n...\n', flush=True)
             run(f'git clone {clone_uri} {chestxray_root}')
+
         transform = transforms.Compose([xrv.datasets.XRayCenterCrop(),
                                         xrv.datasets.XRayResizer(self.hparams.xraysize)])
+
```

Comment:
The variable transform is left unused

Reply:
this was just to replicate their pre processing with kornia. Depending what strategy we follow I could internally send a PR to torchxrayvision and implement kornia there. This two transformations I think are very specific to the dataset. The first, center crops based on the half of the minimum H/W of the image, the second calls resize from skimage to normalise for batching.

Reply:
I agree - let's move to kornia - no problem here.
The transform variable on line is just left unused... let's delete it!